### PR TITLE
Customising Font Display on `Home` Page

### LINKED ISSUE #88: US-16: As a user, I would like the ability to change the text font in the generated summaries, so that I can personalise the appearance of the summary to suit my preferences or readability requirements.
**Tasks:**
- [x] Design and integrate a user interface element for font selection.
- [x] Implement a range of font options for users to choose from.
- [x] Define a default font setting for summaries.
- [x] Develop the logic for rendering selected fonts in summaries. Conduct validation testing to ensure correct font application.
- [x] Prepare for usability testing on font customisation.

**Acceptance Criteria:**
- [x] The user interface element for font selection is visually integrated into the system.
- [x] It is positioned intuitively, ensuring easy discoverability for users.
- [x] Users have access to a variety of font options, including different styles and sizes.
- [x] The provided fonts cover a reasonable spectrum of preferences.
- [x] A default font setting is established for summaries.
- [x] The default font is chosen to be visually appealing and suitable for most users.
- [x] The system successfully renders selected fonts in summaries.
- [x] The font rendering logic accurately applies user-selected fonts to summary text.
- [ ] Validation testing confirms that the selected font is correctly applied to the summary text.
- [ ] Test cases cover various font options and scenarios.
- [ ] Usability testing scenarios and scripts are prepared to evaluate font customization.

### PR DESCRIPTION
**Pull request by gracebilliris**
This Pull Request includes the following tasks from [Jira](https://amlp.atlassian.net/jira/software/projects/SIS/boards/3/backlog?selectedIssue=SIS-124):
- [x] Adding additional text font options: `serif` and `roboto`.
- [x] Designed and integrated a user interface element for font selection. 
- [x] Implementing the range of font options for users to choose from.
- [x] Developed the logic for rendering selected fonts in the `Home` Page.

**UI Screens**
image
image
image

### CODE REVIEW

**Review by srujankarthik — APPROVED** [review]
looks good to me!

File: client/src/pages/Home.tsx [hunk 1]
```diff
@@ -1,26 +1,38 @@
...
 import NavigationBar from "./NavigationBar";
-
 import { addToHistory, clearHistory, postRequestOptions, retrieveHistory } from "../components/Utils";
-import DividerHorizontal from "../components/DividerHorizontal";
 
 function Home(this: any) {
     const [someState, setSomeState] = useState("not set");
-    const [textInput, setTextInput] = useState("Placeholder");
-    const [testTextInput, setTestTextInput] = useState("Insert text here to test analysis storage function");
+    const [textInput, setTextInput] = useState("not set");
```
Comment (stephenb-coding) [inline client/src/pages/Home.tsx:8]: Should probably be initialised as an empty string

**Review by stephenb-coding — APPROVED** [review]
Looks good!

File: client/src/pages/Home.tsx (hunk 1 above)
Comment (gracebilliris) [reply client/src/pages/Home.tsx:8]: It's purposely like this so I could see if the state was updating properly when the input was changing!

**Pull request merged**